packages/react-composites/src/composites/common/ControlBar/CommonCallControlBar.tsx

```diff
@@ -254,6 +263,19 @@ export const CommonCallControlBar = (props: CommonCallControlBarProps & Containe
     !capabilitiesSelector?.capabilities ||
     capabilitiesSelector.capabilities.useReactions.isPresent;
 
+  /* @conditional-compile-remove(breakout-rooms) */
+  const canReturnToMainMeeting = breakoutRoomSettings && breakoutRoomSettings.disableReturnToMainMeeting === false;
+  /* @conditional-compile-remove(breakout-rooms) */
+  const returnFromBreakoutRoom = useMemo(
```

Comment:
Should this be a useCallback instead of a useMemo?

Reply:
Yeah you're right. Changed.